### ISSUE TITLE
Refine RAILScore to accomodate learnings from a inbox trace pair

### ISSUE THREAD

**Issue by natduca** [catapult-project/catapult#1736]
In https://drive.google.com/open?id=0BwIPUR6SMpJVX2U4OExrYzE2cjQ there are two traces of dismissing an item in the inbox webapp: 
- trace_RobotoFalseJsO.json.gz
- trace_RobotoTrueJsO.json.gz

In b/25396149 we conclude that there is relatively little user experience impact on the user. There is a slight efficiency difference in the trace. When you open these in tracing, the railscore for these is about 10% different. But, as we worked through the regression, it was clear that this was a shippable regression: the actual visible impact on the user was zero, and the efficiency went down only slightly.

Ideally, our RAILscore framework would capture this case such that the overall score would be slightly lower in the roboto case, but then we'd see that ux scores were the same, and efficiency was slightly reduced. Such guidance would let people make the right tradeoff.

**Comment by natduca** [comment]
Calling @benshayden

**Comment by benshayden** [comment]
Just trying to clarify what we talked about.

The interesting IR is the invisible missing Response before the Animation at about 2.5 seconds in each trace, right?

In the False trace, it looks like 367ms from ForwardKeyboardEvent to the start of the Animation.
In the True trace, it's 375ms, which is about 2% different from 367.

So the question would be which variant is more CPU- and memory-efficient.
That Idle in the True trace has an efficiency score of 0.487. In False, 0.533, which is about 9% different. But the durations of the True Idle is 1460, and the False Idle 1344, which is about 8% different, so there could be no real efficiency difference.

In order to directly measure the efficiency of those missing Responses, we'd need to fix those keyboard input events (crbug.com/551082), and also trace disabled-by-default-ipc.flow and disabled-by-default-toplevel.flow so that their associatedEvents would be as accurate as currently possible.

Is that right or did I miss something?

Have we talked about whether or not we should trace ipc.flow and toplevel.flow in deepreports/slowreports? Move them to enabled by default? Rewrite Response/Animation associatedEvents using traits ... somehow?
